suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -328,7 +330,9 @@ public void waitForRebuild(long timeout, TimeUnit unit) throws InterruptedExcept
         rebuildLock.lock();
         try {
             while (rebuilding) {
-                rebuildDone.await(timeout, unit);
+                if (!rebuildDone.await(timeout, unit)) {
+                    LOGGER.log(Level.WARNING, "await timed out due to timeout");
```

Comment:
Again, maybe something like:
`Rebuild did not finish in {0} {1}, new Object[] {timeout, unit})`

Reply:
Thanks !